### ISSUE TITLE
Disable `no-string-literal` rule

### ISSUE THREAD

**Issue by blakeembrey** [typings/tslint-config-typings#9]
Sadly it's kind of required to do TypeScript, there's no other way to access object properties that are defined using TypeScript's index types.

**Comment by unional** [comment]
@blakeembrey , can you post an example for reference?

We can easily disable it. 🌷

**Comment by blakeembrey** [comment]
``` ts
interface Env {
  [key: string]: string
}

var x: Env

x.foo // Illegal using TypeScript.
x['foo'] // Legal using TypeScript.
```

Personally the difference using TypeScript has already been confusing why they allow one and not the other since it's just JS, but ignoring that it's the only way to use indexed properties.

**Comment by unional** [comment]
Fixed in 0.3.0
